src/Makefile.am

```diff
@@ -64,10 +65,15 @@ DISPATCH_CFLAGS=-Wall $(VISIBILITY_FLAGS) $(OMIT_LEAF_FP_FLAGS) \
 if DISPATCH_ENABLE_ASSERTS
 DISPATCH_CFLAGS+=-DDISPATCH_DEBUG=1
 endif
+
+if ANDROID
+ANDROID_LIBS= -L$(top_srcdir)/src/shims/android
```

Comment:
@MadCoder now that header overlays are gone I'm even more uncomfortable to having this dummy files (`libpthread.a` and `librt.a`) in `src/shims/android`. I was thinking on `vendors/android/lib(pthread|rt).a` or `deps/android/lib(pthread|rt).a`. What do you think?

More details on why this is required [here](https://github.com/apple/swift-corelibs-libdispatch/pull/162/commits/2b2cb83a1810fe2f8dd37f5c64d0ce00fdcafeda)